administrator/components/com_admin/sql/updates/mysql/6.0.0-2024-08-24.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `ordering`, `state`) VALUES
+(0, 'plg_fields_number', 'plugin', 'number', 'fields', 0, 1, 1, 0, 1, '', '{"min":"1.0","max":"100.0","step":"0.1","currency":"0","position":"0","decimals":"2"}', '', 17, 0);
```

Comment:
```suggestion
(0, 'plg_fields_number', 'plugin', 'number', 'fields', 0, 1, 1, 0, 1, '', '{"min":"1.0","max":"100.0","step":"0.1","currency":"0","position":"0","decimals":"2"}', '', -1, 0);
```
New plugins should be installed on update with order number -1 so they are shown at the top.

Reply:
And the script has to be renamed, see my other comment in the main thread.

Reply:
ok name is changed. Thansk for looking at this.

Reply:
No, the name is not changes. Please check again it is 2024!!!

Reply:
Sorry thought i pushed the change. At the ER with my kid atm waiting. Things are a bit hectic.